app.js

```diff
@@ -103,7 +103,7 @@ app.use('/api', apiRoutes);
 
 /// catch 404 and forward to error handler
 app.use(function(req, res, next) {
-    var err = new Error('Not Found');
+    var err = new Error('404 Not Found: ' + req.url);
```

Comment:
Also, why do we need to show the URL? It is going to be obvious what the URL was.